app/policies/post_policy.rb

```diff
@@ -1,5 +1,13 @@
 class PostPolicy < ApplicationPolicy
-  def owner?
-    user == record.user
+  def edit?
+    owner?
+  end
+
+  def destroy?
+    owner?
+  end
+
+  def new?
+    user != nil
```

Comment:
Ne, šis nebus pareizi!

Skaties.

Tev joprojām ieteiktu pārbaudīt ar `owner?` loģiku, bet iekš new actiona mazliet vēlāk izsaukt authorize!

```ruby
def new
  @comment = @post.comments.build(user: current_user)
  authorize!(@comment) # Šajā vietā jau user ir uzstādīts uz comment, tāpēc viss būs kārtībā!